test/RpcCompatibility.js

```diff
@@ -384,6 +384,33 @@ describe('RpcCompatibility', () => {
         });
     });
 
+    it('should match the fetched logIndex value', async function () {
```

Comment:
```suggestion
    it('should return consistent logIndex values between eth_getLogs and eth_getTransactionReceipt', async () => {
```
need to make the purpose for this specific test clear, and use `async () =>` for consistency